src/client/Microsoft.Identity.Client/Instance/Region/RegionManager.cs

```diff
@@ -160,7 +160,7 @@ private async Task<RegionInfo> DiscoverAndCacheAsync(string azureRegionConfig, I
 
         private async Task<RegionInfo> DiscoverAsync(ICoreLogger logger, CancellationToken requestCancellationToken)
         {
-            string region = Environment.GetEnvironmentVariable("REGION_NAME");
```

Comment:
`toLower` is culture sensitive, please use `toLowerInvariant` https://stackoverflow.com/questions/6225808/string-tolower-and-string-tolowerinvariant